pkgs/development/compilers/crystal/default.nix

```diff
@@ -163,12 +179,13 @@ let
 
       makeFlags = [
         "CRYSTAL_CONFIG_VERSION=${version}"
+        "release=1"
+        "progress=1"
       ];
 
       LLVM_CONFIG = "${llvmPackages.llvm.dev}/bin/llvm-config";
 
       FLAGS = [
-        "--release"
```

Comment:
I guess we no longer need to do this as you're patching the versions that do not support it.

Otherwise good to. Once this is reinstated I'll squash and merge.

Thanks a ton @GeopJr !!!

Reply:
The release flag moved to [`makeFlags`](https://github.com/NixOS/nixpkgs/pull/195606/files#diff-1205fbe1629d11896d8a36ee90fed97d79b6dbdf568bf4201483cdcf62a88318R182) (which has [the same effect as far as I am aware](https://github.com/crystal-lang/crystal/blob/master/Makefile#L33)) from the #173928 PR. Is there another reason to include it in `FLAGS`?